src/main/webapp/WEB-INF/tags/base.tag

```diff
@@ -6,7 +6,7 @@
     </h1>
   </a>
   <div class="searchbox">
-    <div class="gcse-searchbox-only" data-resultsUrl="/search"></div>
+    <div class="gcse-searchbox-only" enableAutoComplete="true" data-resultsUrl="/search"></div>
```

Comment:
I know @kaylaleung mentioned she wasn't able to get the suggestions to show up for her demo. I don't seem to be able to either.

Kayla - were you able to fix it and see suggestions?
Aaron - are you able to see suggestions?

Reply:
Still no suggestions yet :/

Reply:
Got it, could you edit the PR description so that it doesn't say that it fixes #75?

Also, when I looked into this, it seems like this is something you also need to enable from the control panel?
https://blog.expertrec.com/how-to-enable-autocomplete-in-google-custom-search/

But in my control panel, I only see the three search engines for each of the clients and not the actual project CSE. So I unfortunately don't know how to get this to work :'( Not sure if I am missing something/if you're able to figure out the control panel?

Reply:
Autocomplete was enabled (by default) in the control panel since creating the key; adding this attribute was an extra step :')

Reply:
got it, merp yeah i don't have a clue on why it's not working